curriculum/challenges/english/07-scientific-computing-with-python/learn-tree-traversal-by-building-a-binary-search-tree/65ca298b039a852d149f2e95.md

```diff
@@ -1,8 +1,8 @@
 ---
-id: 655a580da8b2419496c88f61
-title: Step 56
+id: 65ca298b039a852d149f2e95
+title: Step 60
 challengeType: 20
-dashedName: step-56
+dashedName: step-60
 ---
 
```

Comment:
In this step the `print("Search for 40:", bst.search(40))` call can be uncommented.